app/models/tslr_claim.rb

```diff
@@ -62,6 +67,11 @@ class TslrClaim < ApplicationRecord
   validates :national_insurance_number, on: [:"national-insurance-number", :submit], presence: {message: "Enter your National Insurance number"}
   validate  :ni_number_is_correct_format
 
+  validates :has_student_loan,                  on: [:"student-loan", :submit], inclusion: {in: [true, false], message: "Select yes if you have a student loan"}
+  validates :student_loan_country,              on: [:"student-loan-country", :submit], presence: {message: "Select the country in which you first applied for your student loan"}
+  validates :student_loan_courses,              on: [:"student-loan-how-many-courses", :submit], presence: {message: "Select the number of higher education courses you have studied"}
```

Comment:
I think there needs to be some conditional validation here. When I select 'Scotland' then try to submit the form I get the following errors:

<img width="668" alt="Screenshot 2019-07-23 at 11 57 57" src="https://user-images.githubusercontent.com/2804149/61707320-1b5c6e80-ad42-11e9-8c9a-665d513e6423.png">

Reply:
Yep, noticed that too, just pushed up a commit that resolves the issue